.github/workflows/check.yaml

```diff
@@ -72,7 +72,6 @@ jobs:
       - name: external controller juju config
         run: |
           mkdir -p $HOME/.local/share/juju
-          # TODO: can we remove the base64 encoding?
```

Comment:
I think we can add a comment about why it's base64 encoded.  I've gone down that rabbit hole before... perhaps in this case it's to avoid issues with templating a multiline string with possible quoting characters, etc. into a shell command?

Reply:
Updated